src/arviz_plots/plots/parallel_plot.py

```diff
@@ -0,0 +1,229 @@
+"""Pair focus plot code."""
+from collections.abc import Mapping, Sequence
+from copy import copy
+from importlib import import_module
+from typing import Any, Literal
+
+import numpy as np
+import xarray as xr
+from arviz_base import dataset_to_dataarray, rcParams
+from arviz_base.labels import BaseLabeller
+
+from arviz_plots.plot_collection import PlotCollection
+from arviz_plots.plots.utils import (
+    filter_aes,
+    get_group,
+    process_group_variables_coords,
+    set_wrap_layout,
+)
+from arviz_plots.visuals import multiple_lines, set_xticks
+
+
+def plot_parallel(
+    dt,
+    var_names=None,
+    filter_vars=None,
+    group="posterior",
+    coords=None,
+    sample_dims=None,
+    plot_collection=None,
```

Comment:
I would add label type as an argument here, and then pass that to `dataset_to_dataarray`. With very few variables vertical mode with horizontal labels looks nice, but we'll generally have more variables in which case I think `flat`+rotation=90 would be better.

I think we should also keep the norm_method argument. Personally I think "rank" is the best method as when diagnosing we don't really care about the actual values, only seeing if diverging and non diverging samples have clear differences in distribution. But I am not strongly opinionated on making it the default.